### ISSUE TITLE
Adding custom user data ASP.NET Core MVC

### ISSUE THREAD

**Issue by laureanray** [aspnet/AspNetCore.Docs#12639]
"The project sample is created from a Razor Pages web app, but the instructions are similar for AN ASP.NET Core MVC web app."  It's hard to follow especially for beginner when using ASP.NET Core MVC. I used CLI tools because I'm using JetBrains Rider and I can't follow all the instructions provided.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 431df4fc-4168-cc8f-d053-9b264d2971c1
* Version Independent ID: cd157d04-d596-83fa-6edb-24d52193eb34
* Content: [Add, download, and delete user data to Identity in an ASP.NET Core project](https://docs.microsoft.com/en-us/aspnet/core/security/authentication/add-user-data?view=aspnetcore-2.2&tabs=visual-studio#feedback)
* Content Source: [aspnetcore/security/authentication/add-user-data.md](https://github.com/aspnet/AspNetCore.Docs/blob/master/aspnetcore/security/authentication/add-user-data.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-security**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
This is not a beginners tutorial.
